.editorconfig

```diff
@@ -14,9 +14,16 @@ indent_size = 2
 [*.py]
 indent_size = 4
 
-# Don't correct spacing for sql and yaml files as sometimes want them wrong for tests
-[*.sql]
```

Comment:
Review this entire PR with whitespace changes hidden:

<img width="425" alt="image" src="https://user-images.githubusercontent.com/10931297/162565137-e3aaa9f4-44cd-4012-9d9c-094d6a40eeab.png">